Makefile

```diff
@@ -93,9 +93,10 @@ distclean: clean
 
 # update docker services
 update: remove-old-images opt dpx-apigateway.env
-	. ./dpx-container-tags && . ./svc.env && export FLUENTD_CONFIG_DIGEST=$(shell date -r ./config/fluent.conf +%s) && export START_DATE=$(shell date --iso-8601=seconds) && $(DOCKER) stack deploy --prune -c dpx.yml dpx --with-registry-auth
+	. ./dpx-container-tags && . ./svc.env && export FLUENTD_CONFIG_DIGEST=$(shell date -r ./config/fluent.conf +%s) && export START_DATE=$(shell date --iso-8601=seconds) && $(DOCKER) system prune -a -f && $(DOCKER) stack deploy --prune -c dpx.yml dpx --with-registry-auth
 
 force-update: remove-old-images opt dpx-apigateway.env
+	$(DOCKER) system prune -a -f
```

Comment:
@apanvalkar should this action be done after the stack has been shutdown - so following the next line which does a wait for all services to exit?

Reply:
@KJMascarenhas  good catch.

Please re-review and merge if all good.

Reply:
I will re-rest once pused to develop on our dev vm.